presto-native-execution/scripts/setup-centos.sh

```diff
@@ -21,6 +21,7 @@ export nproc=$(getconf _NPROCESSORS_ONLN)
 dnf install -y maven
 dnf install -y java
 dnf install -y python3
+dnf install -y python3-devel
```

Comment:
@wypb do we need python3 if we are installing the devel?

Reply:
Hi @majetideepak, thank you for your reply. I removed the `dnf install -y python3` and found this script also works.